packages/runtime/src/templates/edge-shared/rsc-data.ts

```diff
@@ -28,10 +28,9 @@ export declare type PrerenderManifest = {
 
 const noop = () => {}
 
-// Ensure that routes with and without a trailing slash map to different ODB paths
 const rscifyPath = (route: string) => {
   if (route.endsWith('/')) {
-    return route.slice(0, -1) + '.rsc/'
+    return route.slice(0, -1) + '.rsc'
```

Comment:
Have we got a test for this? Do we know why we had the trailing slash in the first place?

Reply:
We do have existing tests for rsc in https://github.com/netlify/next-runtime/blob/main/cypress/e2e/default/appdir.cy.ts 

Overall I did lift RSC related changes to separate standalone PR ( https://github.com/netlify/next-runtime/pull/2290 ) where we can focus on them - I only hit some problems with RSC in this work after I made changes to not use RSC rewrites edge middleware to lambdas (as we finally could vary on RSC header there) which exposed some problems with it